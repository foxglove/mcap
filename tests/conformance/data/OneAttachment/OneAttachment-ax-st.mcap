version https://git-lfs.github.com/spec/v1
oid sha256:64ea30df4433f05f71b1cd621609799fc463a47f88ca8696b08cccca37640335
size 295
