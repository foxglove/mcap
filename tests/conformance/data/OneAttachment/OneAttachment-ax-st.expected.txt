Header library="" profile=""
Attachment content_type=application/octet-stream created_at=1 data=<010203> log_time=2 name=myFile
Statistics attachment_count=1 channel_count=0 channel_message_counts={} chunk_count=0 message_count=0
AttachmentIndex content_type=application/octet-stream data_size=3 length=78 log_time=2 name=myFile offset=25
Footer summary_offset_start=0 summary_start=116
